lib/realtime/block_waiver.ex

```diff
@@ -0,0 +1,68 @@
+defmodule Realtime.BlockWaiver do
+  @moduledoc """
+  A representation of known skipped work on a trip, entered by dispatch.
+  Essentially this summarizes a set of StopTimeUpdate datapoints.
+  """
+
+  alias Concentrate.StopTimeUpdate
+  alias Gtfs.{Block, Trip}
+  alias Realtime.Server
+
+  @type t :: %__MODULE__{
+          trip_id: Trip.id(),
```

Comment:
I agree it might be helpful at some point to process the data further on the server, but given that we are still trying to decide what and how we want to display it on the client I don't know enough yet to decide on different structures.

Reply:
Okay. Then let's keep this in mind as a future change if we ever find working with lots of individual trip waivers annoying, but not do it yet.

Reply:
Satisfied by 60124a1 "Combine block waivers for consecutive skipped stops"